API.md

```diff
@@ -355,43 +355,46 @@ public struct HTTPResponseStatus: Equatable, CustomStringConvertible, Expressibl
 }
 
 /// HTTP Methods handled by http_parser.[ch] supports
-public enum HTTPMethod : RawRepresentable {
-    /* be future-proof, http_parser can be upgraded */
-    case other(String)
+public struct HTTPMethod : RawRepresentable {
```

Comment:
Do we actually gain anything by making this `RawRepresentable`? Seems a little superfluous to me (outside of an enum). Though `Notification.Name` also seems to use it.

Also leads to the duplicate `init` below.